### PR TITLE
Added minizinc

### PR DESCRIPTION
**Pull request by SeppiaBrilla**
Hi, I have added a script to install minizinc and updated the Dockerfile to install minizinc.

### CODE REVIEW

File: etc/build/install-minizinc.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# version as of 16 November 2023
```
Comment (ozgurakgun) [inline etc/build/install-minizinc.sh:3]: looks good! I would just put today's date here in the comment. and checkout 2.8.3 which is the latest tag on github. we don't want to track latest on their main branch - would be bad for reproducibility. some other install-*.sh scripts do this.

File: etc/build/install-minizinc.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+pushd tmp-install-minizinc
+git clone https://github.com/MiniZinc/libminizinc.git
+cd libminizinc
+mkdir build
+cd build
+cmake ..
+cmake --build .
+cp minizinc ${BIN_DIR}/minizinc
+mkdir -p ${BIN_DIR}/share
+cp -r ../share/minizinc ${BIN_DIR}/share/minizinc
```
Comment (ozgurakgun) [inline etc/build/install-minizinc.sh:20]: should the binary be copied to BIN_DIR/minizinc?

File: etc/build/install-minizinc.sh (hunk 1 above)
Comment (SeppiaBrilla) [reply etc/build/install-minizinc.sh]: Makes sense! I have updated the script to do so

File: etc/build/install-minizinc.sh (hunk 2 above)
Comment (SeppiaBrilla) [reply etc/build/install-minizinc.sh:22]: We need to be able to call the binary to use solvers that requires minizinc so, I think is better to have it in the path somewhere

Comment (ozgurakgun) [reply etc/build/install-minizinc.sh:22]: I mean currently it is not getting copied to the BIN_DIR, right?

**Pull request merged**